browser_utils.js

```diff
@@ -312,14 +354,15 @@ async function assertValue(input, expected) {
 }
 
 /**
- * Assert that there is currently no element matching the xpath on the page
- * @param {import('puppeteer').Page} page
- * @param {string} xpath
- * @param {string} [message]
- * @param {number} [wait_ms] 
- * @param {number} [check_every]
+ * Assert that there is currently no element matching the XPath on the page.
+ *
+ * @param {import('puppeteer').Page} page puppeteer page object.
+ * @param {string} xpath XPath to search for.
+ * @param {string?} message Error message shown if the element is not visible in time.
+ * @param {number?} wait_ms How long to wait, in milliseconds.
+ * @param {number?} checkEvery Intervals between checks, in milliseconds.
```

Comment:
more defaults, for wait_ms and checkEvery

Reply:
Added (and renamed `wait_ms` to `waitMs`). Note that we will likely redesign this method to take named arguments, but that's outside the scope of this PR.